### PR TITLE
Added `object-detection` dataset

### PR DESCRIPTION
**Pull request by znym**
Added a dataset for isssue #792

### CODE REVIEW

File: tasks/src/object-detection/data.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-	datasets: [
-		{
-			// TODO write proper description
-			description: "Widely used benchmark dataset for multiple Vision tasks.",
-			id:          "merve/coco2017",
-		},
+		datasets: [
+	  {
+		description: "Widely used benchmark dataset for multiple Vision tasks.",
+		id: "merve/coco2017",
```
Comment (merveenoyan) [inline tasks/src/object-detection/data.ts:7]: I think there are many generic object detection datasets on Hub compared to this :)

File: tasks/src/object-detection/data.ts [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+	  },
+	  {
+		description: "Car Object Detection dataset with annotated bounding boxes around cars.",
```
Comment (merveenoyan) [inline tasks/src/object-detection/data.ts:10]: this one sounds good!

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for this PR! @znym

File: tasks/src/object-detection/data.ts (hunk 2 above)
Comment (znym) [reply tasks/src/object-detection/data.ts:10]: Hi @merveenoyan, thank you so much for the review. Could you please clarify me please if I need to make any changes or what would be the next step I need to follow.

Comment (merveenoyan) [reply tasks/src/object-detection/data.ts:10]: I think there's only above review that you can edit and nothing else for now :)

**Comment by kadirnar** [comment]
@merveenoyan , @znym 

Could you take a look at this for the Coco dataset?

https://huggingface.co/datasets/detection-datasets/coco

**Comment by merveenoyan** [comment]
Hey @znym,
We have moved the codes for Tasks project to [huggingface.js repository](https://github.com/huggingface/huggingface.js) so if you feel like continuing this pull request you can move your work over there. Many thanks! 🤗

**Pull request closed without merging**